code/modules/cargo/packs.dm

```diff
@@ -1604,6 +1609,47 @@
 	                /obj/item/toy/redbutton)
 	crate_name = "toy crate"
 
+/datum/supply_pack/misc/trekuniforms
+	name = "Outdated Uniforms"
+	cost = 400
```

Comment:
Empty crates are worth more than 400, price needs to be at least 700 according to the file comments or cargo can buy this to get free points.